controller/search.go

```diff
@@ -286,6 +293,28 @@ func (c *SearchController) Spaces(ctx *app.SpacesSearchContext) error {
 		q = "" // Allow empty query if * specified
 	}
 
+	var sort string
+	if ctx.Sort == nil {
+		// this is the default case
+		// which returns recently updated spaces first
+		sort = "spaces." + SortByUpdatedAt + " DESC"
+	} else {
+		switch *ctx.Sort {
+		case "name":
+			sort = "spaces." + SortByName + " ASC"
```

Comment:
I think we do not need that prefix "spaces.", If we are using `db.Order()` and [this line](https://github.com/fabric8-services/fabric8-wit/blob/bb6529acba62c7bdeafd10e117e5b2c9c1c0ae32/space/space.go#L280) is setting the model for `db` instance we use later.


just `db.Order(sort)` should suffice IMO.

Reply:
removed!